packages/mdl-button/mdl-button.scss

```diff
@@ -0,0 +1,152 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import "variables";
+
+.mdl-button {
+  display: inline-block;
+  position: relative;
+  min-width: 64px;
+  height: 36px;
+  padding: 0 16px;
+  border: none;
+  border-radius: 2px;
+  outline: none;
+  background: none;
+  color: rgba(0, 0, 0, .87);
+  font-family: Roboto, sans-serif;
+  font-size: .875rem; /* 14sp */
```

Comment:
Good catch here, the spec defines it as `pt` specifically which we would translate to a native pixel. So let's move forward with that (need to go whitelist pixels as well for font-size which means we're really going to need to watch ourselves in the future here.)

However, we should probably ask for MD guidance on if we should adapt to users font sizes here. If a user asks for a larger fonts, our buttons should scale accordingly to fit their needs. If someone does need a 2xl font size, they shouldn't be stuck with regular button text they may not be capable of reading easily.